src/js/gi/components/search/SearchResult.jsx

```diff
@@ -92,7 +92,6 @@ export class SearchResult extends React.Component {
       </div>
     );
   }
-
```

Comment:
Super nitpick: if removing this newline at the end of the class, can also remove the one at the top.